src/Symfony/Component/Config/Tests/Builder/Fixtures/ArrayValues/Symfony/Config/ArrayValues/ErrorPagesConfig.php

```diff
@@ -44,7 +44,7 @@ public function withTrace($value): static
      * @param array{ // Default: {"enabled":false}
      *     enabled?: bool, // Default: false
      *     with_trace?: bool,
-     * } $config
+     * }|bool $config
```

Comment:
there's a mismatch with the type of the argument, but the shape is good
and this doesn't matter because I'm going to move shapes out of generated configurators in a PR later today

Reply:
I'll remove the shape (move it elsewhere)